docs/HACKING.md

```diff
@@ -212,41 +211,143 @@ opm index add --bundles $BUNDLE_REPOSITORY:$BUNDLE_TAG --tag $INDEX_REPOSITORY:$
 ```
 
 You should then push the newly created index image to the remote repository:
+
 ```shell script
 podman push $INDEX_REPOSITORY:$INDEX_TAG
 ```
 
 #### Editing an existing operator index
+
 An existing operator index can have bundles added to it:
+
 ```shell script
 opm index add --from-index $INDEX_REPOSITORY:$INDEX_TAG
 ```
+
 and removed from it:
+
 ```shell script
 opm index rm --from-index $INDEX_REPOSITORY:$INDEX_TAG
 ```
+#### Create a CatalogSource using the index
+
+See the OpenShift docs found
+[here](https://docs.openshift.com/container-platform/latest/operators/admin/olm-managing-custom-catalogs.html#olm-creating-catalog-from-index_olm-managing-custom-catalogs)
+
+#### Create a subscription object 
+
+To use the custom CatalogSource you need to change the subscription object to reference the CatalogSource you are 
+introducing. See the docs 
+[here](https://docs.openshift.com/container-platform/4.5/operators/understanding/olm/olm-understanding-olm.html#olm-subscription_olm-understanding-olm)
+for an example subscription. 
+
+Edit your subscription yaml to match your CatalogSource. 
+For example, 
+```
+  source: < new catalogSource > 
+  sourceNamespace: < namespace where CatalogSource was deployed >
+```
+
+Apply the yaml to your cluster, which will deploy the WMCO. 
+
+## Deploying using the official deployment method 
```

Comment:
Please move this above `Build`

Reply:
since this section is related to deploying the WMCO, I think it makes sense to keep it here where the other deployment methods are. I get how deploying using the official method is different than deploying using the other methods, but even if you don't have to build anything to deploy using this, we should keep the deployment methods in the same place.

Reply:
I don't agree. By keeping it here you are giving the impression that a build needs to be done to use the official deployment method.